modules/experimental/pages/portal.adoc

```diff
@@ -0,0 +1,145 @@
+However, there are some important differences that are necessary to maintain game balance.
+
+== Data sheet variations of {portal}
+
+Where we include {portal} on a unit data sheet -- or for an objective -- we also include some further information, as attributes in brackets.
+For example, like this: `{portal} _(INF, 4+, {walker}, Auto)_`.
+
+Mostly, these attributes indicate what other friendly units can actually pass through the portal -- that is, which kinds of unit that portal permits you to embark and/or disembark as 'cargo' units during the game.
+
+* `_(INF)_`: Permits Infantry units as cargo.
+* `_(4+)_`: Permits Vehicle units that have an Armour value of `4+` or lower as cargo.
+* `_({walker})_`: Permits Vehicle units that have the {walker} ability as cargo, regardless of their Armour value.
+
+We also indicate how the portal affects the arrival of detachments that you intend to use with it, through the `_(Roll)_` or `_(Auto)_` attributes.
+See <<Portal Reserves>> for more about this.
+
+TIP: We use `_(Roll)_` and `_(Auto)_` to represent different technologies for different factions.
+They are mutually exclusive, for both thematic and practical reasons -- so all {portal} units in a given army list are of one type, never a mix of both.
+
+== Game effects of {portal}
+
+Where your army includes any units or objectives that have the {portal} ability:
+
+* Regardless of the scenario, you may keep some detachments out of the play area, as <<Portal Reserves>>.
+You may do this with any number of your detachments that contain units that your {portal} units permit as cargo.
++
+TIP: Don't get carried away -- if you have many Portal Reserves detachments out of play and few suitable {portal} units in play then you're in for a hard time!
+
+Where your unit or objective has the {portal} ability and is in play:
+
+* You can use this unit to move Portal Reserves detachments into play after they arrive.
+See <<Portal Reserves>>.
+* You can use this unit to move detachments that are in play from this portal to another portal that is in play, or from another portal that is in play to this portal.
+See <<portal-moves>>.
+
+[[portal-moves]]
+== Use a {portal} unit to move your forces
+
+You can use portals to move some of your forces around.
+Where you wish to move one of your detachments into or out of a portal, and it contains permitted cargo units:
+
+* You may do this only in the Movement phase.
+* You may do this only if you give the detachment Advance or Assault orders.
+See xref:main-rules:transportation.adoc[].
+footnote:[
+When you disembark cargo units from an *objective* that has the {portal} ability, you obviously cannot pay 5 cm of its movement like you would with a mobile transport unit.
+In this case, just deduct 5 cm from the movement of each cargo unit that you disembark, as usual, and ignore the deduction that would normally apply to the transport unit.
+]
```

Comment:
I'm not sure why I put the Transportation topic reference in the same paragraph as the bit about Advance or Assault orders — that seems odd, because it implies that that topic has something to say about those orders, which it doesn't. 

I probably meant to put such a link _somewhere_ to reference the rules for embarking and disembarking with with the {transport-x} ability.

```suggestion
* You may do this only in the Movement phase, and only if you give the detachment Advance or Assault orders.
* Embark and disembark cargo units in the same way as for the {transport-x} ability.
See xref:main-rules:transportation.adoc#_transportation_by_transport_x_units[Transportation by Transport (X) units].
footnote:[
When you disembark cargo units from an *objective* that has the {portal} ability, you obviously cannot pay 5 cm of its movement like you would with a mobile transport unit.
In this case, just deduct 5 cm from the movement of each cargo unit that you disembark, as usual, and ignore the deduction that would normally apply to the transport unit.
]
```